shared/actions/chat/inbox.js

```diff
@@ -143,12 +143,12 @@ function* onInboxStale(param: Constants.InboxStale): SagaGenerator<any, any> {
     const inboxIsEmpty = I.Map(conversations.map(c => [c.conversationIDKey, c.isEmpty]))
 
     yield all([
-      put(EntityCreators.replaceEntity(['inboxVersion'], idToVersion)),
-      put(EntityCreators.replaceEntity(['inboxSmallTimestamps'], inboxSmallTimestamps)),
+      put(EntityCreators.replaceEntity([], I.Map([['inboxVersion', idToVersion]]))),
```

Comment:
Maybe these should just be function calls that live somewhere common? Seems like a complicated call that could be easy to get wrong.

Reply:
these only ever happen here